api/src/test/resources/bigquery/data/concept_data.json

```diff
@@ -0,0 +1,7 @@
+[
```

Comment:
I'm wondering if these belong in some other dir; we're going to need them when loading up BQ outside of tests.

Also, we should differentiate between OMOP tables and your own custom tables (maybe in the dir name or something.)

Reply:
I'm fine with that. Should we just wait till to make a decision on that when we are actually loading these files outside of tests? Also, these json files don't build the entire table schema for efficiency of the test cases. They only create columns that are currently needed for testing.

Reply:
Yeah, fine to defer this to later.

FWIW, I suspect the difference between creating all columns and just the columns needed for tests will be negligible... we should probably in future switch to using the full schema.